Version_testNet

```diff
@@ -1,4 +1,4 @@
-DOCKER_IMAGE_FUSE_APP_VERSION="2.0.0"
+DOCKER_IMAGE_FUSE_APP_VERSION="2.0.4"
 DOCKER_IMAGE_FUSE_PARITY_VERSION="3.0.0"
-DOCKER_IMAGE_NET_STATS_VERSION="1.0.0"
-DOCKER_IMAGE_NM_CLIENT="nethermind-1.13.3-v3.0.1"
\ No newline at end of file
+DOCKER_IMAGE_NET_STATS_VERSION="2.0.1"
+DOCKER_IMAGE_NM_CLIENT="nethermind-1.25.3-v4.0.5-alpha"
```

Comment:
Shouldn't this also be using nethermind-1.25.3-v4.0.5 (not the alpha, I know they are the same thing just to avoid any confusion)